ci/cpu/build.sh

```diff
@@ -49,16 +56,16 @@ conda config --set ssl_verify False
 # BUILD - Conda & pip package
 ################################################################################
 
-logger "Build conda pkg for frigate..."
-conda build conda/recipes/frigate --python=$PYTHON
+gpuci_logger "Build conda pkg for frigate"
+gpuci_conda_retry build conda/recipes/frigate --python=$PYTHON
 
-logger "Build pip pkg for frigate..."
+gpuci_logger "Build pip pkg for frigate"
 rm -rf dist/
 python setup.py sdist bdist_wheel
```

Comment:
This can be removed, we don't build pip packages anymore.

Reply:
Completed